### DISCUSSION TITLE
New PR failed

### DISCUSSION THREAD

**Discussion by ritamgit-alt** [mindsdb/mindsdb#2111] [Get help]
why I'm not able to make a new pull request!
![32EBB090-A58C-4775-A09E-579694CAFCD8_1_201_a](https://user-images.githubusercontent.com/93112390/161706566-2fdbc9d9-a29d-41bf-9791-d2e06621ddb9.jpeg)
!

**Comment by ZoranPandovski** [comment]
Answered #2110
